Dockerfile.sdk

```diff
@@ -29,12 +29,16 @@ ARG BASE_IMAGE=nvidia/cuda:10.1-devel-ubuntu18.04
 
 ARG TRITON_COMMON_REPO_TAG=main
 ARG TRITON_CORE_REPO_TAG=main
+ARG TRITON_MODEL_ANALYZER_REPO_TAG=main-dev
 
 FROM ${BASE_IMAGE}
 
 # Ensure apt-get won't prompt for selecting options
 ENV DEBIAN_FRONTEND=noninteractive
 
+# DCGM version to install for Model Analyzer
+ENV DCGM_VERSION=2.0.13
```

Comment:
We may have to hold off on this for now. If we go with NVML, then all of this new installation is not needed.